### PR TITLE
Some bug fixing and enhancements

### PR DESCRIPTION
**Pull request by PerrineGilloteaux**
@miura @czhci could you validate?

### CODE REVIEW

**Comment by PerrineGilloteaux** [comment]
Please include the last 2 commits as well d97b6bf and af67871 in your review.

File: .gitignore [hunk 1]
```diff
@@ -48,6 +48,7 @@ web/sites/simpletest
 private
 .env
 .env-template
+.lando.yml
```
Comment (miura) [inline .gitignore]: .lando.yml file should not be listed in .gitignore. This file is used by Lando to start up the server and needs to be the same among all developers. Please delete this line!

**Comment by PerrineGilloteaux** [comment]
Hi Kota ,
Yes but if someone start lando with a different name for example , it will be different for developers . And in addition this file is created by lando init, locally for the developper, there is no reason to edit it for all developers, is there?

**Comment by miura** [comment]
> Hi Kota , Yes but if someone start lando with a different name for example , it will be different for developers . And in addition this file is created by lando init, locally for the developper, there is no reason to edit it for all developers, is there?

@PerrineGilloteaux , it's free to change the name of lando instance but this then needs additional changes in other files to keep the Drupal running. Changes in the local is better be limited to minimal (such as env file, setting files). The trouble @czhci faced last month when she was trying to run the lando instance in her laptop, was actually this problem (now I understand). 

Kota

**Comment by PerrineGilloteaux** [comment]
Hi Kota ,I ve edited the .gitignore

**Comment by miura** [comment]
Great, could you also add back lando.yml? You can find the original in
Github.

**Comment by PerrineGilloteaux** [comment]
Hi @miura ,lando.yml is back.
Note that the comment bug is fixed in drupal 8.6.10 (see https://github.com/NeuBIAS/bise/issues/69 ), I will commit it and do a new PR, once this one is deployed.

**Pull request merged**